ext/pathname/pathname.c

```diff
@@ -1248,6 +1249,24 @@ path_opendir(VALUE self)
     return rb_block_call(rb_cDir, id_open, 1, args, 0, 0);
 }
 
+/*
+ * Changes the current path to the referenced directory.
+ *
+ * See Dir.chdir.
+ */
+static VALUE
+path_chdir(VALUE self)
+{
+    VALUE args[1];
+
+    if(rb_block_given_p()) {
```

Comment:
Very Minor: is the `if (` the style used in other conditionals in this file?

Reply:
Oh, indeed! Thank you for noticing.